config/locales/diaspora/en.yml

```diff
@@ -1315,7 +1311,6 @@ en:
       hide_and_mute: "Hide and mute post"
       like: "Like"
       unlike: "Unlike"
-      dislike: "Dislike"
```

Comment:
I wonder if this string is really not needed anymore… I couldn’t find it used anywhere, but there are references to dislikes in *lib/diaspora/likeable.rb* and *spec/controllers/likes_controller_spec.rb*.